### PR TITLE
Workaround for buggy Intel graphics drivers

### PR DESCRIPTION
**Pull request by pmconne**

### CODE REVIEW

File: core/webgl-compatibility/src/test/Compatibility.test.ts [hunk 1]
```diff
@@ -111,63 +114,99 @@ describe("Render Compatibility", () => {
...
+      [ "ANGLE (Intel(R) UHD Graphics 630 Direct3D11 vs_5_0 ps_5_0)", true ],
+      [ "ANGLE (Intel(R) HD Graphics 620 Direct3D11 vs_5_0 ps_5_0)", true ],
+      [ "ANGLE (Intel(R) UHD Graphics 620 Direct3D11 vs_5_0 ps_5_0)", true ],
+      [ "ANGLE (Intel HD Graphics 620 Direct3D11 vs_5_0 ps_5_0)", false ],
+
+      // Bug only confirmed on 620 and 630
+      [ "ANGLE (Intel(R) UHD Graphics 610 Direct3D11 vs_5_0 ps_5_0)", false ],
+      [ "ANGLE (Intel(R) UHD Graphics 615 Direct3D11 vs_5_0 ps_5_0)", false ],
+      [ "ANGLE (Intel(R) UHD Graphics 500 Direct3D11 vs_5_0 ps_5_0)", false ],
+      [ "ANGLE (Intel(R) UHD Graphics 520 Direct3D11 vs_5_0 ps_5_0)", false ],
```
Comment (DStradley) [inline core/webgl-compatibility/src/test/Compatibility.test.ts:169]: UHD only applies 600, so may be best to make these two HD

Comment (pmconne) [reply core/webgl-compatibility/src/test/Compatibility.test.ts:169]: [The bug](https://dev.azure.com/bentleycs/beconnect/_workitems/edit/484001) contains a screenshot from the user in which unmasked renderer string contains "UHD Graphics 630".

Comment (pmconne) [reply core/webgl-compatibility/src/test/Compatibility.test.ts:169]: Ah, I see what you're saying. The test is checking to make sure we *don't* detect these as false positives. They don't have to match real renderes. I'll add "HD".

**Review by mgooding — APPROVED** [review]

File: core/webgl-compatibility/src/test/Compatibility.test.ts (hunk 1 above)
Comment (DStradley) [reply core/webgl-compatibility/src/test/Compatibility.test.ts:169]: Yes, I meant to type UHD only applies to 6xx (at least so far), not 5xx.

File: core/frontend/src/render/webgl/TechniqueFlags.ts [hunk 2]
```diff
@@ -43,10 +43,13 @@ export const enum IsThematic { No, Yes }
  * @internal
  */
 export class TechniqueFlags {
+  /** System will set this to true upon initialization if necessary. */
+  public static requireEdgeTest = false;
```
Comment (pmconne) [inline core/frontend/src/render/webgl/TechniqueFlags.ts:47]: @mgooding @DStradley I'm not in love with this solution but I wanted to ensure we always apply the edge test (force surface discard) without having to add conditionals all over the place.

**Review by mgooding — APPROVED** [review]

**Review by ramanujam-raman — APPROVED** [review]

**Comment by calebmshafer** [comment]
@Mergifyio backport release/2.9.0

**Pull request merged**